massa-node/base_config/config.toml

```diff
@@ -11,6 +11,8 @@
     bind_private = "127.0.0.1:33034"
     # port on which the node API listens for public requests. Can be exposed to the Internet
     bind_public = "0.0.0.0:33035"
+    # port on which the node API listens for HTTP requests and WebSOckets subscriptions. Can be exposed to the Internet
+    bind_api = "0.0.0.0:33036"
```

Comment:
Maybe rename it to v2 to be more clear for users.

Reply:
Just add it in the comment could be fine, I don't went to add v2 everywhere

Reply:
Adding it in config file that users reads doesn't seems too much to me but you can just put a comment ok

Reply:
Done